src/typ.rs

```diff
@@ -2002,23 +2002,37 @@ This should not be possible. Please report this crash",
                 };
 
                 // Insert unqualified imports into scope
-                for UnqualifiedImport { name, meta } in &unqualified {
+                for UnqualifiedImport {
+                    name,
+                    meta,
+                    as_name,
+                } in &unqualified
+                {
                     let mut imported = false;
 
+                    let unqualified_alias = match &as_name {
+                        None => name.clone(),
+                        Some(alias) => alias.clone(),
```

Comment:
Does this compile if we remove the `.clone()` from these two lines? It would be good to only clone the string once (when we insert it) rather than twice.

Reply:
Yes it does seem to compile. I've changed it!